src/main/java/com/wolox/training/security/CustomAuthenticationProvider.java

```diff
@@ -0,0 +1,31 @@
+package com.wolox.training.security;
+
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+
+@Component
+public class CustomAuthenticationProvider implements AuthenticationProvider {
+
+    @Override
+    public Authentication authenticate(Authentication authentication)
+            throws AuthenticationException {
+
+        String name = authentication.getName();
```

Comment:
You should authenticate with the database user password, not with a server level password

Reply:
Solved, i integrated the authentication service with a validation of users in database.